pyresample/kd_tree.py

```diff
@@ -510,12 +492,10 @@ def _create_resample_kdtree(source_lons,
         raise EmptyResult('No valid data points in input data')
 
     # Build kd-tree on input
-    if kd_tree_name == 'pykdtree':
-        resample_kdtree = KDTree(input_coords)
-    elif nprocs > 1:
+    if nprocs > 1:
```

Comment:
I think this case can be removed too

Reply:
We should probably remove the `nprocs` option all together then. Right?

Reply:
yep